### ISSUE TITLE
[Fix Included!] Using `@import()` doesn't work in Better Discord

### ISSUE THREAD

**Issue by steenburgh** [CroissantDuNord/discord-adblock#11]
TL;DR, try using this instead
```css
@import url("https://steenburgh.github.io/discord-adblock/adblock.css");
```

# The Issue
If you paste the `@import()` line from this repo's README in the Better Discord custom CSS window, the script doesn't load + an error shows up in the dev tools. Copy/pasting the full raw script in works. I suspect some recent change GitHub made broke this - possibly to prevent `raw.githubusercontent.com/whatever` from being used for web hosting. I haven't tested this with Venicord, but wouldn't be surprised if it has a similar issue if I'm right about the cause

![image](https://github.com/CroissantDuNord/discord-adblock/assets/9411156/072fcf2b-7d63-4b6a-8f4c-0b0603580e43)
![image](https://github.com/CroissantDuNord/discord-adblock/assets/9411156/956cfccc-d41a-43d0-a5a8-6461c0d4bb98)

# The Fix
I have a fix for this - I forked the repo + set up GitHub pages on the fork. Then I used the GitHub pages URL of the CSS file + this worked perfectly! I'd recommend the maintainer of this repo do the same :).
https://github.com/steenburgh/discord-adblock

![image](https://github.com/CroissantDuNord/discord-adblock/assets/9411156/e26b497c-3687-4591-a593-acc31142ad4b)
![image](https://github.com/CroissantDuNord/discord-adblock/assets/9411156/41eefc64-b299-4a3c-82b8-b93e9f131193)

**Comment by CroissantDuNord** [comment]
This could be Github making change strangely Vencord still support raw.githubusercontent.com, But if I say support for BetterDiscord I will so I switched to Github Pages.

https://croissantdunord.github.io/discord-adblock/adblock.css
